README.md

```diff
@@ -80,6 +80,12 @@ Role Variables
     # Config map for zombie slots pruner
     zombie_slots_pruner_rcm
 
+    # remote host auth secret for zombee slots pruner
+    zombie_slots_pruner_remote_host_secret
+
+    # remote host auth mount path for zombee slots pruner
+    zombie_slots_pruner_remote_host_mount_path
```

Comment:
might deserve a rename, we're not mounting remote hosts, just their auth keys

Reply:
added 'auth' there